runtime/polkadot/src/weights/pallet_bounties.rs

```diff
@@ -50,101 +50,59 @@ impl<T: frame_system::Config> pallet_bounties::WeightInfo for WeightInfo<T> {
 	// Storage: Bounties Bounties (r:0 w:1)
 	/// The range of component `d` is `[0, 16384]`.
 	fn propose_bounty(d: u32, ) -> Weight {
-		// Minimum execution time: 29_378 nanoseconds.
-		Weight::from_ref_time(32_175_061 as u64)
-			// Standard Error: 18
-			.saturating_add(Weight::from_ref_time(829 as u64).saturating_mul(d as u64))
-			.saturating_add(T::DbWeight::get().reads(2 as u64))
-			.saturating_add(T::DbWeight::get().writes(4 as u64))
+		// Minimum execution time: 29_384 nanoseconds.
+		Weight::from_ref_time(31_061_881)
+			// Standard Error: 7
+			.saturating_add(Weight::from_ref_time(819).saturating_mul(d.into()))
+			.saturating_add(T::DbWeight::get().reads(2))
+			.saturating_add(T::DbWeight::get().writes(4))
 	}
-	// Storage: Bounties Bounties (r:1 w:1)
-	// Storage: Bounties BountyApprovals (r:1 w:1)
 	fn approve_bounty() -> Weight {
-		// Minimum execution time: 12_073 nanoseconds.
-		Weight::from_ref_time(12_863_000 as u64)
-			.saturating_add(T::DbWeight::get().reads(2 as u64))
-			.saturating_add(T::DbWeight::get().writes(2 as u64))
+		// Minimum execution time: 0 nanoseconds.
+		Weight::from_ref_time(0)
 	}
-	// Storage: Bounties Bounties (r:1 w:1)
 	fn propose_curator() -> Weight {
-		// Minimum execution time: 11_317 nanoseconds.
-		Weight::from_ref_time(11_559_000 as u64)
-			.saturating_add(T::DbWeight::get().reads(1 as u64))
-			.saturating_add(T::DbWeight::get().writes(1 as u64))
+		// Minimum execution time: 0 nanoseconds.
+		Weight::from_ref_time(0)
```

Comment:
Some of the (child)bounties for Polkadot and Rococo are indeed going down as predicted here https://github.com/paritytech/substrate/pull/12829#issuecomment-1342440386  
[bounties.pdf](https://github.com/paritytech/polkadot/files/10246530/bounties.pdf)  
I only dont understand why the `spend_funds` have -97% :thinking:.  
Will take a closer look later.